tedana/metrics/kundu_fit.py

```diff
@@ -362,36 +363,45 @@ def kundu_metrics(comptable, metric_maps):
     Generate three metrics of component noise:
     - countnoise: Number of "noise" voxels (voxels highly weighted for
       component, but not from clusters)
-    - signal-noise_t: T-statistic for two-sample t-test of F-statistics from
+    - countsignal: Number of "signal" voxels (voxels highly weighted for
+      component, from clusters)
+    - signal-noise_z: Z-statistic for two-sample t-test of F-statistics from
       "signal" voxels (voxels in clusters) against "noise" voxels (voxels not
       in clusters) for R2 model.
     - signal-noise_p: P-value from t-test.
     """
     comptable['countnoise'] = 0
-    comptable['signal-noise_t'] = 0
+    comptable['signal-noise_z'] = 0
     comptable['signal-noise_p'] = 0
     for i_comp in comptable.index:
         # index voxels significantly loading on component but not from clusters
-        comp_noise_sel = ((np.abs(Z_maps[:, i_comp]) > 1.95) &
+        comp_noise_idx = ((np.abs(Z_maps[:, i_comp]) > 1.95) &
                           (Z_clmaps[:, i_comp] == 0))
-        comptable.loc[i_comp, 'countnoise'] = np.array(
-            comp_noise_sel, dtype=np.int).sum()
-        # NOTE: Why only compare distributions of *unique* F-statistics?
-        noise_FR2_Z = np.log10(np.unique(F_R2_maps[comp_noise_sel, i_comp]))
-        signal_FR2_Z = np.log10(np.unique(
-            F_R2_maps[Z_clmaps[:, i_comp] == 1, i_comp]))
-        (comptable.loc[i_comp, 'signal-noise_t'],
-         comptable.loc[i_comp, 'signal-noise_p']) = stats.ttest_ind(
+        comptable.loc[i_comp, 'countnoise'] = int(np.sum(comp_noise_idx))
+        noise_FR2_Z = 0.5 * np.log(F_R2_maps[comp_noise_idx, i_comp])
+        signal_FR2_Z = 0.5 * np.log(F_R2_maps[Z_clmaps[:, i_comp], i_comp])
+        n_noise_dupes = noise_FR2_Z.size - np.unique(noise_FR2_Z).size
```

Comment:
`dupes` makes me think of being `duped`. Can we change to something like `dupl` ?